app/models/Phase5Generators.scala

```diff
@@ -83,7 +81,7 @@ object Phase5Generators extends TemplateArgGenerators {
     "limitDate"                        -> dateFormatter.format(limitDate)
   )
 
-  val transitOperationType04: ArgGen = for {
+  val TransitOperationType04: ArgGen = for {
```

Comment:
vals generally start with lowercase but can see why you have used upper!

Reply:
Yes, simply for readability and compatibility with the XSD! :)